OrbitGl/App.cpp

```diff
@@ -645,17 +662,7 @@ void OrbitApp::OnLoadCapture(const std::string& file_name) {
   string_manager_->Clear();
   thread_pool_->Schedule([this, file_name]() mutable {
     capture_loading_cancellation_requested_ = false;
-    ErrorMessageOr<void> result =
-        capture_deserializer::Load(file_name, this, &capture_loading_cancellation_requested_);
-
-    if (result.has_error()) {
-      if (open_capture_failed_callback_) {
-        open_capture_failed_callback_();
-      }
-      SendErrorToUi("Error loading capture",
-                    absl::StrFormat("Could not load capture from \"%s\":\n%s", file_name,
-                                    result.error().message()));
-    }
+    capture_deserializer::Load(file_name, this, &capture_loading_cancellation_requested_);
 
     if (open_capture_finished_callback_) {
       open_capture_finished_callback_();
```

Comment:
Should this be called from OnCaptureComplete?

Reply:
Well that basically calls `loading_capture_dialog->close();`. Even though it does not hurt if this is called also for normal captures, it would feel a bit strange. Also we need to call this here for either case (successful load, error, cancellation).

Reply:
I moved this anyway.